### DISCUSSION TITLE
domino/14.0.0/admin/inst_dock_domino_overview

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#83] [Q&A]
# domino/14.0.0/admin/inst_dock_domino_overview

HCL Domino is available as a container image that can be used with current Docker or Podman runtimes. HCL Domino is preinstalled on this image, allowing you to deploy Domino servers as containers with just a few commands.

https://help.hcl-software.com/domino/14.0.0/admin/inst_dock_domino_overview.html

**Comment by elenko-db** [comment]
Wondering about the Domino 14 docker image - it never appeared in the HCLtech download site. The Domino 12 and 11 docker images are still available, and actually the 14.5 early access one too. Is there a special reason for this?

**Reply by gthiel-id** [reply]
Hi - thanks, this site is for feedback on the documentation content itself, but for this I'll forward that to development and let you know what they say

**Reply by gthiel-id** [reply]
What I found out is that, depending on licensing, you might not still be entitled to the docker image for v14? - For v14, the HCL Complete Collaboration Solutions edition doesn't include that, nor does Domino Enterprise or Domino Messaging. My contact verified that the image appears for the HCL Complete Collaboration Business edition.

**Comment by Daniel-Nashed** [comment]
Even the container image is not available for all customers depending on the license type, you can use the community container build environment.
The [**HCL Domino community container project**](https://opensource.hcltechsw.com/domino-container/) provides an easy and flexible way to build images including all add-on applications you are entitled for.

The project also as an end to end integration downloading software from [**My HCL Software portal (MHS)**](https://my.hcltechsw.com/)

For questions about the community image, refer to the documentation or open a GitHub issue.
